entity-services/src/test/java/com/marklogic/entityservices/tests/TestEntityTypes.java

```diff
@@ -131,8 +131,10 @@ private void checkXMLRoundTrip(String message, Document original, Document actua
         invalidMessages.put("invalid-namespace-blacklist.xml", "It is a reserved pattern.");
         invalidMessages.put("invalid-namespace-blacklist2.json", "It is a reserved pattern.");
         invalidMessages.put("invalid-namespace-blacklist2.xml", "It is a reserved pattern.");
-        invalidMessages.put("invalid-namespace-dupprefix.json", "You cannot duplicate a namespace prefix within a model descriptor");
-        invalidMessages.put("invalid-namespace-dupprefix.xml", "You cannot duplicate a namespace prefix within a model descriptor");
+        invalidMessages.put("invalid-namespace-dupprefix.json", "Each prefix and namespace pair must be unique.");
+        invalidMessages.put("invalid-namespace-dupprefix.xml", "Each prefix and namespace pair must be unique.");
+        invalidMessages.put("invalid-namespace-dupuri.json", "Each prefix and namespace pair must be unique.");
+        invalidMessages.put("invalid-namespace-dupuri.xml",  "Each prefix and namespace pair must be unique.");
         invalidMessages.put("invalid-namespace-noprefix.json", "If a type has a namespace property, it must also have a namespacePrefix property.");
         invalidMessages.put("invalid-namespace-noprefix.xml", "If a type has a namespace property, it must also have a namespace-prefix property.");
```

Comment:
Looks like all cases are covered. Just want to mention that the cases for "noprefix" or "noouri" will give following error message 
`"If a type has a namespace-prefix property, it must also have a namespace property. Each prefix and namespace pair must be unique."`
if these models had another type with proper prefix and uri. Additional message might throw users off and look for issue in wrong places. What do you think?

Reply:
ill look into it. having all those messages seems not the right thing to do

Reply:
It is true that if a model fails with one rule, it also fails with the other
I can work one more change in that will prevent this misleading message.